### PR TITLE
workflows: add jobs for container builds

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

**Comment by lucab** [comment]
This is green on my fork: https://github.com/lucab/fedora-coreos-cincinnati/actions/runs/349274986

File: .github/workflows/containers.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  build-fedora-infra:
+    name: "Build container image (fedora-infra)"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build container image (fedora-infra)
+        uses: docker/build-push-action@v2
+        with:
+          push: false
```
Comment (kelvinfan001) [inline .github/workflows/containers.yml:20]: Doesn't `push` [default to `false`](https://github.com/marketplace/actions/build-and-push-docker-images#customizing) already?

**Review by kelvinfan001 — APPROVED** [review]
LGTM

File: .github/workflows/containers.yml (hunk 1 above)
Comment (lucab) [reply .github/workflows/containers.yml]: Good catch! I dropped it then.

**Pull request merged**